tests/openvino/test_quantization.py

```diff
@@ -849,6 +849,14 @@ def test_config_from_dict(self, quantization_config: dict, config_type: type, wa
             if hasattr(ov_config.quantization_config, k):
                 self.assertEqual(getattr(ov_config.quantization_config, k), v)
 
+    @parameterized.expand(_DEFAULT_4BIT_CONFIGS)
```

Comment:
Could you also please include `_DEFAULT_4BIT_CONFIG` here?

Reply:
I'll do that, but the test wouldn't look acceptable then, since `_DEFAULT_4BIT_CONFIGS` is a dictionary with the `model_id` as key and `_DEFAULT_4BIT_CONFIG` is not.

Reply:
this will not work?
@prametrized(_DEFAULT_4BIT_CONFIGS.update({"default": _DEFAULT_4BIT_CONFIG))

Reply:
Unfortunately, no. `.update` method doesn't return any value.